flink-core/src/main/java/org/apache/flink/configuration/NettyShuffleEnvironmentOptions.java

```diff
@@ -184,12 +184,13 @@
 					"tasks have occupied all the buffers and the downstream tasks are waiting for the exclusive buffers. The timeout breaks" +
 					"the tie by failing the request of exclusive buffers and ask users to increase the number of total buffers.");
 
-	@Documentation.ExcludeFromDocumentation("This option is only used for testing at the moment.")
 	public static final ConfigOption<String> NETWORK_BOUNDED_BLOCKING_SUBPARTITION_TYPE =
 		key("taskmanager.network.bounded-blocking-subpartition-type")
```

Comment:
rename to `taskmanager.network.blocking-shuffle.type` to unify it with the resolution of the discussion around the compression configs? And mention that it's experimental?

If we rename it, it should probably be mentioned in the release notes. Maybe it was excluded from the docs, but we told about this property on the mailing list to some users.

Reply:
Since this options was not exposed to users before, so it has no compatibility concern when renaming it, I agree with this suggestion.